src/backend/distributed/sql/citus--10.0-3--10.1-1.sql

```diff
@@ -24,3 +24,21 @@ ALTER TABLE pg_catalog.pg_dist_rebalance_strategy ADD COLUMN improvement_thresho
 UPDATE pg_catalog.pg_dist_rebalance_strategy SET improvement_threshold = 0.5 WHERE name = 'by_disk_size';
 
 #include "udfs/get_rebalance_progress/10.1-1.sql"
+
+-- use streaming replication when replication factor = 1
+WITH replicated_shards AS (
+    SELECT shardid
+    FROM pg_dist_placement
+    WHERE shardstate = 1
```

Comment:
`OR shardstate = 3`?

Reply:
We can potentially have a shard that is marked for deletion after a rebalance operation. Do you think it is safer to assume we have replication factor>1 in that case?

I am ok either way.

Reply:
Marked for deletion is shardstate=4. The number of placements in shardstate=4 says nothing about the replication factor, but shardstate=3 indicates there is replication.